File: SampleSetup/Protocol/Model/ApiCountry.cs
using AbstractModelWithId = SampleProject.Protocol.Model.AbstractModelWithId;
using ApiCity = SampleProject.Protocol.Model.ApiCity;
using ApiAddress = SampleProject.Protocol.Model.ApiAddress;
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Model
{
    public class ApiCountry : AbstractModelWithId
    {
        public string TitleShort { get; set; }

        public string TitleLong { get; set; }

        public string PhoneCode { get; set; }

        public string Code { get; set; }

        public List<ApiCity> Cities { get; set; } = new List<ApiCity>();

        public List<ApiAddress> Addresses { get; set; } = new List<ApiAddress>();

    }
}

File: SampleSetup/Output/PHP_8/Model/ApiCountry.php
<?php
namespace SampleProject\Protocol\Model;

class ApiCountry extends \SampleProject\Protocol\Model\AbstractModelWithId {
    public string $TitleShort;

    public string $TitleLong;

    public string $PhoneCode;

    public string $Code;

    /**
     * @var \SampleProject\Protocol\Model\ApiCity[]
     */
    public $Cities;

    /**
     * @var \SampleProject\Protocol\Model\ApiAddress[]
     */
    public $Addresses;

    public function __construct()
    {
        parent::__construct();
        $this->Cities = [];
        $this->Addresses = [];
    }

}
